Knossos.NET/Views/GlobalSettingsView.axaml

```diff
@@ -143,6 +143,7 @@
 				</Border>
 				<!--VIDEO SETTINGS-->
 				<Label Margin="5,15,0,0" FontWeight="Bold" FontSize="18" ToolTip.Tip="Video settings the FSO engine will use in-game.">Video</Label>
+				<TextBox BorderBrush="Transparent" IsReadOnly="True" TextWrapping="Wrap" Margin="5,0,5,0" Foreground="Yellow">As of FSO version 24.2 video settings not longer have any effect and is only intended to use with old versions. On 24.2 or newer you can change settings ingame by going to "Options" and pressing F3 to display all settings.</TextBox>
```

Comment:
`Note, video settings for most mods using FSO version 24.2.0 or higher will need to be changed in-game by default. You can change in-game settings by going to "Options" menu of the mod you are playing.`

I've written an alternate version you could use based on my previous comments.